app/models/template.rb

```diff
@@ -146,6 +145,32 @@ def import_oses(options)
     end
   end
 
+  def import_taxonomies(options)
+    process_organizations options
+    process_locations options
+  end
+
+  def process_organizations(options)
+    process_taxonomies options, :organization
+  end
+
+  def process_locations(options)
+    process_taxonomies options, :location
+  end
+
+  def process_taxonomies(options, taxonomy)
+    tax_options = options["#{taxonomy}_params".to_sym]
+    if tax_options.empty?
+      send("import_#{taxonomy}s", options)
```

Comment:
How about `#{taxonomy.pluralize}` just in case we ever another taxonomy with irregular pluralization rules (and to be consistent with the rest of the code)